perl_lib/EPrints/Extras.pm

```diff
@@ -304,7 +304,7 @@ sub render_possible_doi
 
 	$value = "" unless defined $value;
 	if( $value =~ m!^
-			(?:http://(?:dx\.)?doi\.org/)?  # add this again later anyway
+			(?:http(?:s)://(?:dx\.)?doi\.org/)?  # add this again later anyway
```

Comment:
Hi, I would be even less restrictive here. According to [RFC 1808](http://www.ietf.org/rfc/rfc1808.txt) we could use simply // unless explicit use of https or http. That means that if the user server is in https, //dx.doi.org will lead to https://dx.doi.org, while, if browsing in http, we will have http://dx.doi.org